websites/D/Duolingo/presence.ts

```diff
@@ -1,296 +1,277 @@
-/* eslint-disable no-one-time-vars/no-one-time-vars */
 const presence = new Presence({
-		clientId: "909577563234508910",
+		clientId: "1177802176140156998",
 	}),
-	FLAG_ICONS = [
-		"ar",
-		"ca",
-		"cs",
-		"cy",
-		"da",
-		"de",
-		"dn",
-		"el",
-		"en",
-		"eo",
-		"es",
-		"fr",
-		"ga",
-		"gn",
-		"he",
-		"hi",
-		"hu",
-		"hv",
-		"hw",
-		"id",
-		"it",
-		"ja",
-		"kl",
-		"ko",
-		"nb",
-		"nv",
-		"pl",
-		"pt",
-		"ro",
-		"ru",
-		"sv",
-		"sw",
-		"tr",
-		"uk",
-		"vi",
-		"zs",
-	],
-	INFO_PAGES = [
-		"approach",
-		"contact",
-		"cookies",
-		"efficacy",
-		"guidelines",
-		"info",
-		"mobile",
-		"privacy",
-		"team",
-		"terms",
-	],
-	API_ENDPOINTS = [
-		"2017-06-30",
-		"api",
-		"friendships",
-		"login",
-		"switch_language",
-		"users",
-		"vocabulary",
-	],
+	timeStamp = newTimeStamp(),
+	LANGUAGE_NAMES: Record<string, string> = {
+		ar: "Arabic",
+		ca: "Catalan",
+		cs: "Czech",
+		cy: "Welsh",
+		da: "Danish",
+		de: "German",
+		el: "Greek",
+		en: "English",
+		eo: "Esperanto",
+		es: "Spanish",
+		fi: "Finnish",
+		fr: "French",
+		gd: "Scottish Gaelic",
+		gn: "Guarani",
+		he: "Hebrew",
+		hi: "Hindi",
+		ht: "Haitian Creole",
+		hu: "Hungarian",
+		hv: "High Valyrian",
+		hw: "Hawaiian",
+		id: "Indonesian",
+		it: "Italian",
+		ja: "Japanese",
+		ko: "Korean",
+		la: "Latin",
+		dn: "Dutch",
```

Comment:
????, is this actually the name for dutch?

Reply:
Yeah, took me such a long time to debug that one..
For some reason Duolingo gave Dutch 3 different 'codes'.

![Screenshot_20231126-124719.png](https://github.com/PreMiD/Presences/assets/73395916/05c1d9cd-4920-41de-8ba3-b181618f0f21)

![Screenshot_20231126-124737.png](https://github.com/PreMiD/Presences/assets/73395916/698f0933-53c5-4258-a631-92d0f9c360b9)

Reply:
interesting.

Reply:
Mostly they use iso 639-1 (two letter: de, la, etc..), otherwise they use locales when necessary. When locales are involved though, they use their own type of language code on some parts of the site. Which makes things quite confusing.

Will be looking into it, and adding support for the primitive codes soon.
For now I only implemented a "fix" for Dutch, since that's where I found the issue on.

Reply:
Fixed: 5f552c6d61f1cb4edf8f774dd2773d23e2008669